modules/v2x/v2x_proxy/app/utils.h

```diff
@@ -0,0 +1,120 @@
+/******************************************************************************
+ * Copyright 2020 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+/**
+ * @file utils.h
+ * @brief utils for v2x_proxy
+ */
+
+#pragma once
+
+#ifndef __cplusplus
+#error "This is a c++ header!"
+#endif
+
+#include <memory>
+#include <set>
+#include <string>
+#include <unordered_set>
+#include <vector>
+
+#include "cyber/cyber.h"
+#include "modules/map/hdmap/hdmap.h"
+#include "modules/map/hdmap/hdmap_common.h"
+#include "modules/perception/proto/traffic_light_detection.pb.h"
+#include "modules/planning/proto/planning.pb.h"
+#include "modules/v2x/v2x_proxy/proto_adapter/proto_adapter.h"
+
+#ifndef __NEED_OUTPUT_TRAFFIC_LIGHT_WITHOUT_V2X
+#define __NEED_OUTPUT_TRAFFIC_LIGHT_WITHOUT_V2X (true)
+#endif
+
+namespace v2x {
+namespace proxy {
+const char *const UNKNOWN_JUNCION_ID = "unknown";
```

Comment:
Should  be `const std::string& kUnknownJunctionId = "unknown"; `
1) prefer C++ notation over C
2) Coding style

Reply:
"apollo.sh lint" will report:
modules/v2x/v2x_proxy/app/utils.h:43:  For a static/global string constant, use a C style string instead: "const char kUnknownJunctionId[]".  [runtime/string] [4]